tools/stress_mcp.py

```
@@ -0,0 +1,270 @@
+                        writer.close()
+                        await writer.wait_closed()
+
+                        read_obj = json.loads(resp.decode("utf-8", errors="ignore"))
+                        result = read_obj.get("result", read_obj) if isinstance(read_obj, dict) else {}
+                        if not result.get("success"):
+                            stats["apply_errors"] = stats.get("apply_errors", 0) + 1
+                            await asyncio.sleep(0.5)
+                            continue
+                        data_obj = result.get("data", {})
+                        contents = data_obj.get("contents") or ""
+                    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)